app/views/merchants/merchant_orderitems.html.erb

```diff
@@ -0,0 +1,76 @@
+<section class="h-center-text-no-pad">
+  <h2>Merchant's Order Items</h2>
+  <h4>Total Revenue: $<%= '%.2f' % Merchant.calculate_total_revenue(@current_merchant) %></h4>
+</section>
+
+<section class="table-container">
+<section>
+  <div class="summary">
+    <h3>Unfulfilled Orders</h3>
+    <p><span class="bolded">Total Revenue: $</span><%= '%.2f' % Merchant.calc_rev_by_status(@current_merchant, "false") %></p>
+    <p><span class="bolded">Number of Orders: </span><%= Merchant.find_merchants_orderitems(@current_merchant, "false").length %></p>
+  </div>
+
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Order Number</th>
+        <th>Product</th>
+        <th>Date Paid</th>
+        <th>Cost x Quantity</th>
+        <th>Subtotal</th>
+        <th>Status</th>
+        <th>Shipping</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% Merchant.find_merchants_orderitems(@current_merchant, "false").each do |orderitem| %>
```

Comment:
re: Elizabeth's request for focused feedback: Could this logic be simplified by making use of the instance variable orderitems, i.e. `@current_merchant.orderitems`